### DISCUSSION TITLE
Updating script

### DISCUSSION THREAD

**Discussion by DancingEmptier667** [t3nk3y/palia_steam_helper#2] [Q&A] [answered]
Hello!
I already have the script installed. How can I update it? Do I just need to manually download "palia_steam_helper.sh" from the repository?

**Comment by madalee-com** [comment, marked as answer]
Just run the same command as you did on first install.  It should detect
the already existing install, update the script file, and quit.



If you are running a much older version, you won't have a Steam folder in
the script directory, then you can't directly update your setup to the
current version, and the easiest way to handle things would be to let it
start a new install in a new folder.
